index.js

```diff
@@ -1,9 +1,10 @@
 (() => {
     const STATES = { ANSWER: "answer", QUESTION: "question" }
+    const ANSWER = { RIGHT: "right", WRONG: "wrong" }
     const $content = document.getElementById("content");
-    const $thumbsGroup = document.getElementById("buttonAction");
-    const $thumbsUp = document.getElementById("thumbsUp");
-    const $thumbsDown = document.getElementById("thumbsDown");
+    const $question = document.getElementById("question");
+    const $answer_wrong = document.getElementById("answer_wrong");
```

Comment:
```suggestion
    const $answerWrong = document.getElementById("answer__wrong");
```
JS Naming convertion: camelCase (e.q. wrongPassword, invoiceHeader)